src/main/java/com/javadiscord/jdi/internal/api/DiscordRequestBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.javadiscord.jdi.internal.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DiscordRequestBuilder {
```

Comment:
this is not a builder make it a builder

Reply:
Here's something to think about:

So the class isn't actually following the builder pattern, it's just named builder but it's more so just a class that contains chained setters/getters. 

The reason I can't make it a builder per-say, is because I don't have a good class name to give to the "product" that the builder will then create. `DiscordRequest` clashes with the `interface DiscordRequest`